src/main/resources/application.yml

```diff
@@ -229,22 +229,14 @@ spring:
 ---
 
 document-template:
-  part-a-template-settings[0].start-date-time: "2025-01-10T15:58Z"
-  part-a-template-settings[0].template-name: "NAT Recall Part A London Template - obtained 231114.docx"
-  part-a-template-settings[1].start-date-time: "2025-07-30T23:00Z" # 2025-07-31 00:00 in BST
-  part-a-template-settings[1].template-name: "NAT Recall Part A London Template - obtained 231114 - v2025-07-29.docx"
-  part-a-template-settings[2].start-date-time: "2025-08-07T23:00Z" # 2025-08-08 00:00 in BST
-  part-a-template-settings[2].template-name: "NAT Recall Part A London Template - obtained 231114 - v2025-08-07.docx"
-
-  part-a-preview-template-settings[0].start-date-time: "2025-01-10T15:58Z"
-  part-a-preview-template-settings[0].template-name: "Preview NAT Recall Part A London Template - obtained 231114.docx"
-  part-a-preview-template-settings[1].start-date-time: "2025-07-30T23:00Z" # 2025-07-31 00:00 in BST
-  part-a-preview-template-settings[1].template-name: "Preview NAT Recall Part A London Template - obtained 231114 - v2025-07-29.docx"
-  part-a-preview-template-settings[2].start-date-time: "2025-08-07T23:00Z" # 2025-08-08 00:00 in BST
-  part-a-preview-template-settings[2].template-name: "Preview NAT Recall Part A London Template - obtained 231114 - v2025-08-07.docx"
-
-  dntr-template-settings[0].start-date-time: "2025-01-31T14:00Z"
-  dntr-template-settings[0].template-name: "DNTR Template.docx"
+  # Any setting not specified or occurring only in the future will default to the 'default' template name
+  # part-a-template-settings[x].start-date-time/template-name # Settings for Part A templates
+  # part-a-preview-template-settings[x].start-date-time/template-name # Settings for Part A Preview templates
+  # dntr-template-settings[x].start-date-time/template-name # Settings for DNTR templates
+  part-a-template-settings[0].start-date-time: "2025-09-01T23:00Z" #2025-09-02T00:00Z in BST
+  part-a-template-settings[0].template-name: "2025-09-02 - FTR48 Phase 1"
+  part-a-preview-template-settings[0].start-date-time: "2025-09-01T23:00Z" #2025-09-02T00:00Z in BST
```

Comment:
The preview config values are now obsolete, aren't they?

Reply:
I think this is the merge conflict no resolving the way I thought it did. I am going to pull latest and manually check this. Same for the rest of the comments below on this block.